src/shared/views/plugins/DeepZoom/DeepZoomRoute.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
```

Comment:
Mixes Route and Redux. Not sure if this is best way to do it. But otherwise I would be creating a route that just has a container than then has the same stuff. Extra layer, didn't seem to be a good reason for it.